### PR TITLE
fix: Night tickets should be presented on Recent Purchases view

### PR DESCRIPTION
**Pull request by jorelosorio**
Mentioned on https://github.com/AtB-AS/kundevendt/issues/3068

This PR shows night tickets on recent purchases.

Screenshot_1673276259

### CODE REVIEW

File: src/screens/Ticketing/FareProducts/use-recent-fare-contracts.ts [hunk 1]
```diff
@@ -98,6 +100,32 @@ const mapBackendRecentFareContracts = (
...
+  );
+
+  if (fareProductTypeConfig.configuration.zoneSelectionMode === 'none') {
+    return {
+      preassignedFareProduct,
+      fromTariffZone: undefined,
+      toTariffZone: undefined,
+      userProfilesWithCount,
+    };
+  }
```
Comment (gorandalum) [inline src/screens/Ticketing/FareProducts/use-recent-fare-contracts.ts:127]: Should also check `userProfilesWithCount.length` before returning?

File: src/screens/Ticketing/FareProducts/use-recent-fare-contracts.ts [hunk 2]
```diff
@@ -107,11 +135,6 @@ const mapBackendRecentFareContracts = (
...
   );
 
-  const userProfilesWithCount = mapUsers(
-    recentFareContract.users,
-    userProfiles,
-  );
-
   if (
     !preassignedFareProduct ||
     !fromTariffZone ||
```
Comment (gorandalum) [inline src/screens/Ticketing/FareProducts/use-recent-fare-contracts.ts:140]: Maybe the check on `preassignedFareProduct` and `userProfilesWithCount?.length` is redundant here as it is already checked further up?

**Review by gorandalum — CHANGES_REQUESTED** [review]
Looks good. Just a small check that I think should be in place so we don't get any surprises later on.

**Review by rosvik — APPROVED** [review]
LGTM 👍

File: src/screens/Ticketing/FareProducts/use-recent-fare-contracts.ts (hunk 2 above)
Comment (jorelosorio) [reply src/screens/Ticketing/FareProducts/use-recent-fare-contracts.ts]: Fixed now, please take a look ;)

**Review by gorandalum — APPROVED** [review]

**Comment by HanneLH** [comment]
Tested [through](https://github.com/AtB-AS/kundevendt/issues/3068)

**Pull request merged**